pkg/kubelet/metrics/metrics.go

```diff
@@ -917,6 +921,28 @@ var (
 			StabilityLevel: metrics.ALPHA,
 		},
 	)
+
+	// DRAPrepareResourcesDuration tracks the duration of the DRA PrepareResources request.
+	DRAPrepareResourcesDuration = metrics.NewHistogram(
```

Comment:
Are you sure you don't want a histogramVec here? You may want to add labels in the future.